### PR TITLE
Add check for double-F3 to move validation & util file

### LINKED ISSUE #29: Prevent player from creating two free-threes in one move
- [x] See title
- [x] Except if player is capturing a pair

### PR DESCRIPTION
**Pull request by ekelen**
- checks for F3s created on any axis
- adds axis constants
- fix createPlayer test with extra param in test file
- comment out broken CheckSequence benchmark test
- add `*sandbox*` to gitignore
- vscode to gitignore
- add board util and add to unit tests
- some benchmark tests also
- many tests

To test:
- `go run cmd/gomoku/main.go` --> no index out of bounds errors
- `go test -v ./...` --> should be all good

Closes #29

### CODE REVIEW

File: internal/brain/free_three.go [hunk 1]
```diff
@@ -66,3 +79,212 @@ func CheckSequenceForF3(sequence []int16, playerId int16) []int16 {
...
+
+		switch axis {
+		case NS:
+			whichSeq = int16(position % board.SIZE)
+			len = int16(board.SIZE)
+			startGobanI = whichSeq
+			step = int16(board.SIZE)
+			seqOffset = int16((position - startGobanI) / step)
+
+			if checkAxisForF3_2(
```
Comment (eskombro) [inline internal/brain/free_three.go]: This function call is exactly the same in any of the 4 directions. Could we take it out of every case and do it once at the end of the checks?

File: internal/brain/free_three.go [hunk 2]
```diff
@@ -66,3 +79,212 @@ func CheckSequenceForF3(sequence []int16, playerId int16) []int16 {
...
+			whichSeq = board.SIZE - row + col - 1
+			len = whichSeq + 1
+			if whichSeq > board.SIZE-1 {
+				len = (board.SIZE - 1) - whichSeq
+			}
+			if len < 5 {
+				break
+			}
+			step = board.SIZE + 1
+			startGobanI = 342 - (board.SIZE * whichSeq)
```
Comment (eskombro) [inline internal/brain/free_three.go]: What's 342?

File: internal/brain/free_three.go [hunk 3]
```diff
@@ -66,3 +79,212 @@ func CheckSequenceForF3(sequence []int16, playerId int16) []int16 {
...
+			if whichSeq >= 18 {
```
Comment (eskombro) [inline internal/brain/free_three.go]: 18 -> SIZE -1?

File: internal/brain/free_three.go [hunk 4]
```diff
@@ -66,3 +79,212 @@ func CheckSequenceForF3(sequence []int16, playerId int16) []int16 {
...
+				startGobanI = whichSeq - 18
```
Comment (eskombro) [inline internal/brain/free_three.go]: 18 -> SIZE -1?

File: internal/brain/free_three.go (hunk 1 above)
Comment (ekelen) [reply internal/brain/free_three.go]: It can indeed.

File: internal/brain/free_three.go (hunk 2 above)
Comment (ekelen) [reply internal/brain/free_three.go]: I will name it

File: internal/brain/free_three.go (hunk 3 above)
Comment (ekelen) [reply internal/brain/free_three.go]: I will name it (18 is half the total number of sequences)

**Review by eskombro — DISMISSED** [review]

**Review by eskombro — APPROVED** [review]
:)

**Pull request merged**